src/shared/components/case-editor/case-edit-page/case-edit-page.component.ts

```diff
@@ -306,4 +303,11 @@ export class CaseEditPageComponent implements OnInit, AfterViewChecked {
     }
     return caseEventData;
   }
+
+  private setFocusToTop() {
+    const topContainer = document.getElementById('top');
```

Comment:
Toolkit should not be dependant on the consumer implementing ID's, however we seem to have done this already in the past.

Reply:
Agreed - I am aiming to add functionality which removes the dependency on the consumer implementing IDs, so this is only an intermediary change.